redfish-core/lib/memory.hpp

```diff
@@ -820,7 +1082,32 @@ inline void requestRoutesMemory(App& app)
             return;
         }
 
-        getDimmData(asyncResp, dimmId);
+        getDimmObject(asyncResp, dimmId, getDimmData);
+        });
+
+    BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/Memory/<str>/")
+        .privileges({{"Login"}})
```

Comment:
let's use the privilege framework 

redfish::privileges::getMemory

Reply:
Thank you for your valid comments. We have created the JIRA ticket https://jsw.ibm.com/browse/PFEBMC-620 to address this comment.
The scope of this task was to cherry pick the working HWIsolation commits from 1030. And as the number of commits are more, it is difficult to refactor them to align with upstream as part of this PR. So we will address all them as a separate story.